doc/src/manual/interfaces.md

```diff
@@ -565,11 +565,12 @@ end
 find_aac(bc::Base.Broadcast.Broadcasted) = find_aac(bc.args)
 find_aac(args::Tuple) = find_aac(find_aac(args[1]), Base.tail(args))
 find_aac(x) = x
+find_aac(x::Base.Broadcast.Extruded) = x.x
```

Comment:
Should this recurse?  That is, should it actually be this?  I'm honestly not sure.

```suggestion
find_aac(x::Base.Broadcast.Extruded) = find_aac(x.x)
```

Reply:
Might as well. It looks like I actually did this in my library https://github.com/SymbolicML/DynamicQuantities.jl/blob/9625403f76385603a3acd0fa1119fada3912a5ba/src/arrays.jl#L285